content/S2016/square-event.md

```diff
@@ -0,0 +1,13 @@
+title: Square Tech Workshop: Becoming Ok with Okhttp
+Date: 2016-05-25 
+Category: Events 
+Tags: academic 
```

Comment:
I'd say the tags "industry", "workshop", and "open source" more accurately describe this event :) You can take a look at all the current tags attached to posts at http://wics.uwaterloo.ca/tags/ .